app/components/quick-filter.js

```diff
@@ -2,6 +2,13 @@ import Component from '@ember/component';
 import moment from 'moment';
 
 export default Component.extend({
+  didInsertElement: () => {
```

Comment:
This is not the ember way, either use a form element to trigger an event or semantic UI callbacks, don't use jquery and don't add any additional selectors like `id='searchevents'.

Reply:
ok